### DISCUSSION TITLE
How to update to new release ?

### DISCUSSION THREAD

**Discussion by Lojs** [devmfc/debian-on-amlogic#27] [Q&A] [answered]
Hi, Frist i want to thank you for your amazing work. 
Second, Is there anyway to update to new releases without losing any data ?

**Comment by devmfc** [comment, marked as answer]
Depends what you mean by “update to new release’. 

You can update Debian/Ubuntu just by using `apt-get update && apt-get upgrade`. I never tried to upgrade to a newer Debian/Ubuntu release.. might work, I don’t know. 

To upgrade the kernel, you can get the kernel package from a newer release, put it on the box and run `dpkg -i ./linux-image-6.6.16-meson64_20240212_arm64.deb`. But keep a USB flash drive with a working image on the side in case it doesn’t work as expected. I know one of the older versions had a small bug causing an kernel upgrade not to update the config properly, but I don’t remember which one, sorry.

All other stuff I add to the images, that is not part of the standard Debain/Ubuntu releases, are only there to boot the boxes / add new boxes and is not worth updating I think.

**Reply by petboy69** [reply]
but what i can download from here is an .img file not a debiab package
